app/models/case_contact_report.rb

```diff
@@ -9,6 +9,7 @@ def filtered_case_contacts(args)
     CaseContact
       .supervisors(args[:supervisor_ids])
       .creators(args[:creator_ids])
+      .casa_org(args[:casa_org_id])
```

Comment:
@Thrillberg thanks for taking a look at this! We actually don't want to be able to pass in a different casa org (an admin for a particular casa org shouldn't ever be able to get info for a different organization). Instead I think you want to take your casa_org scope and apply it, something like:

```
CaseContact
  .joins(:casa_case).where(casa_cases: { casa_org: current_organization })
  .supervisors(...)
  .creators(...)
 ...
```

Reply:
Thanks for your help on this @littleforest! I'm a little confused about how the `CaseContactReport` would have access to the `current_organization` variable. My understanding was that `current_organization` is available to controllers and from the controller would have to be passed into any collaborators like this.

If you look at the `CaseContactReportsController` change I made, the `casa_org_id` is always the current organization's id and is not changeable from the front end. (That is the intent of my code anyway. Whether it actually works that way may be questionable! 😆)

Reply:
Whoops, @Thrillberg you are totally right! Great. As long as the `casa_org_id` from the front-end is not changeable (or hackable) then all is good.